src/custom/components/swap/SwapModalHeader/SwapModalHeaderMod.tsx

```diff
@@ -109,7 +115,7 @@ SwapModalHeaderProps) {
                 fontWeight={500}
                 color={showAcceptChanges && trade.tradeType === TradeType.EXACT_OUTPUT ? theme.primary1 : ''}
               >
-                {trade.inputAmount.toSignificant(6)}
+                {formatSmart(trade.inputAmount, DEFAULT_PRECISION)}
```

Comment:
nit: Maybe we should have our own formatSmart that has default precission by default and delegate? 
This might remove some repetition and also would make it easier to change the logic in all the places at once?

Reply:
but we do, take a look inside `utils/format` I am wrapping dex-js formatSmart. I chose to keep the option for precision as it may be useful later

Reply:
https://github.com/gnosis/cowswap/pull/895/files#diff-5e3ddbfbe5376cda8fb7e0ae4ac8de168317a2182dbdf93d0445ddec6e302953R18

Reply:
Ahh, right. Still a nit, but if u set `DEFAULT_PRECISION` as the default for `decimalsToShow`, you remove most of the second arguments. 

![image](https://user-images.githubusercontent.com/2352112/124969212-b9789a80-e026-11eb-873c-416d9b2cff2a.png)

Reply:
@anxolin done 544bac2